src/bot.js

```diff
@@ -16,14 +16,15 @@ const retweet = () => {
     q: config.query,
     result_type: config.result_type,
     lang: config.lang,
-    tweet_mode: "extended"
+    tweet_mode: 'extended'
   };
 
   TwitterBot.get('search/tweets', params, (err, data) => {
     // when no errors
     if (!err) {
-      if (data.statuses[0].full_text.split('#').length - 1 === 1) {
+            if ((data.statuses[0].full_text.length <140 ) && (data.statuses[0].full_text.split('#').length - 1 === 1) && (data.statuses[0].full_text.toLowerCase().includes("#100daysofcode"))){
```

Comment:
```suggestion
            if ((data.statuses?.[0]?.full_text?.length < 140 ) && (data.statuses?.[0]?.full_text?.split('#')?.length - 1 === 1) && (data.statuses?.[0]?.full_text?.toLowerCase()?.includes("#100daysofcode"))){
```

Reply:
I do not know how robust the API is with defaults, but seeing so many accessors without optional chaining is scary.